src/main/java/igym/services/GymService.java

```diff
@@ -101,4 +101,19 @@ public Gym findById(UUID id) {
         return gym;
     }
 
+    public List<Gym> findGymsByUserId(UUID userId) {
+        logger.info("Fetching gyms for user with id: {}", userId);
+    
+        userService.findById(userId);
```

Comment:
It took time to me to understand why are you calling `findById`, but not storing the return of the call.
I guess, you are silent validating if the user exists, right? I suggest to express this validation clearer. In order to smooth the maintenance
```suggestion
    if (!userService.existsById(userId)) {
        logger.warn("User with id {} not found", userId);
        throw new UserNotFoundException("User with id " + userId + " not found");
    }
```

Reply:
If you want to keep the `findById` method, I suggest you to leave a comment explaining the intent of this method there